format/view-spec.md

```diff
@@ -75,13 +77,17 @@ Notes:
 
 Each version in `versions` is a struct with the following fields:
 
-| Requirement | Field name        | Description                                                              |
-|-------------|-------------------|--------------------------------------------------------------------------|
-| _required_  | `version-id`      | ID for the version                                                       |
-| _required_  | `schema-id`       | ID of the schema for the view version                                    |
-| _required_  | `timestamp-ms`    | Timestamp when the version was created (ms from epoch)                   |
-| _required_  | `summary`         | A string to string map of [summary metadata](#summary) about the version |
-| _required_  | `representations` | A list of [representations](#representations) for the view definition    |
+| Requirement | Field name          | Description                                                                   |
+|-------------|---------------------|-------------------------------------------------------------------------------|
+| _required_  | `version-id`        | ID for the version                                                            |
+| _required_  | `schema-id`         | ID of the schema for the view version                                         |
+| _required_  | `timestamp-ms`      | Timestamp when the version was created (ms from epoch)                        |
+| _required_  | `summary`           | A string to string map of [summary metadata](#summary) about the version      |
+| _required_  | `representations`   | A list of [representations](#representations) for the view definition         |
+| _optional_  | `default-catalog`   | Catalog name to use when a reference in the SELECT does not contain a catalog |
+| _optional_  | `default-namespace` | Namespace to use when a reference in the SELECT is a single identifier        |
```

Comment:
This should not be optional.

Reply:
Updated to required and added clarification for how to handle this when it is null or missing.

Reply:
thanks